src/app/machines/views/MachineList/MachineList.tsx

```diff
@@ -37,6 +37,33 @@ const parseFilters = (filters: Filters): FetchFilters => {
   return fetchFilters;
 };
 
+const ErrorsNotification = ({
+  errors,
+  onAfterDismiss,
+}: {
+  errors: APIError;
+  onAfterDismiss?: () => void;
+}) => {
+  const [isOpen, setIsOpen] = useState(true);
+  const previousErrors = usePrevious(errors);
+  const handleDismiss = () => {
+    setIsOpen(false);
```

Comment:
I suppose this could remove the request from the store instead as we no longer need the request (a new one will get created when another search is performed) but it will get cleaned up later if this doesn't do it.

Reply:
I wanted to avoid having to interact with the redux store directly here and additional complexity. Like you said, this will get cleaned up anyway.